examples/ads.amp.html

```diff
@@ -1459,6 +1460,14 @@ <h2>Pressboard</h2>
       data-media="5">
   </amp-ad>
 
+  <h2>PromoteIQ</h2>
+  <amp-ad width="250" height="250"
+      type="promoteiq"
+      data-src="https://cdn.tagdelivery.com/ad/client/standard.js"
+      data-params='{"slot": 1160, "targets": { "category": ["6222", "16158", "274", "17912"]}, "count": 2}'
+      data-sfcallback='function (response) { return response;}'>
```

Comment:
Oh also, @aisleypay do you think this is possibly a security issue by allowing the user to embed / run their own JS?

cc @lannka

Reply:
Each of our clients have their own CDN. To enable our clients who want to use PromoteIQ on AMP-enabled pages, they have to pass that specific CDN, which includes client specific PromoteIQ logic. 

The `sfcallback` is for client specific rendering. Each client can render an ad in any way they please. Our clients take our responses and render the responses in their desired format. In order for PromoteIQ to work on AMP enabled pages, the ability to allow clients to still use their custom rendering logic is key.

Reply:
@aisleypay Ah, thank you for the reason! That makes sense, but it definitely opens up a lot of security / Developer experience on the AMP side of things. Thus, I think it would be good to get a response from @lannka and/or @cramforce

Reply:
interesting, so you don't mind if the client is manipulating the ads or even fake clicks?

anyway, I don't see a security issue to AMP pages as we are rendering it in an iframe sandbox. It's more ad network's choice.

Reply:
@lannka Can you clarify what you mean by 'manipulating the ads'? 

As far as fake clicks go, we work closely with our clients on inventory implementation - QA and constant monitoring. Impression/click capture and monitoring is part of the PromoteIQ integration process and for the lifetime of our relationship with the client. When we see suspicious behavior we work with our clients to resolve it.

Reply:
Got you. I guess you are probably doing more reservation ads, so you don't need to worry about pub modifying the ad contents to have click fraud.

Reply:
@lannka @torch2424 Indeed! Do either of you have any further questions or concerns per this thread?